### ISSUE TITLE
Do not set the dask cache to 1/10th of RAM on import

### ISSUE THREAD

**Issue by GenevieveBuckley** [napari/napari#2329]
## 🐛 Bug
Copy-pasting this over from a discussion in dask-image

@jni says https://github.com/dask/dask-image/issues/194#issuecomment-786450341
>  yes napari sets the dask cache to 1/10th of RAM on import. Yes I know it's bad, I've tried to argue to change that.

@tlambert03 https://github.com/dask/dask-image/issues/194#issuecomment-786611351
> Wait which part is bad? I don't remember any arguments about this (and is it really only 1/10 now?)

@jni
https://github.com/dask/dask-image/issues/194#issuecomment-786968555
> Well, it seems to me that modifying the behaviour of a popular third party library (dask) just by importing is undesirable. We see above that just importing napari altered the behaviour of dask during diagnostics. In general, imho there should be a visible function call to alter the dask caching behaviour.
> 
> At any rate, we discussed this in various group meetings, just never seriously. The basic issue I've run into is that the dask cache is good-ish when exploring, but bad when using "play", because you fill up the cache and then spend the whole time putting stuff into the cache queue and deleting it from the back of it.
> 
> > (and is it really only 1/10 now?)
> 
> [napari/napari#1666](https://github.com/napari/napari/pull/1666)

and then @tlambert03 says
> totally agree. let's change it!

## To Reproduce

https://github.com/dask/dask-image/issues/194 is an example of a problem popping up because of this

**Comment by tlambert03** [comment]
hi @GenevieveBuckley, can you summarize the crux of the problem?  I tried to read through that issue to see why the cache was causing that issue... but I didn't immediately see it.  I see that @jni lamented our cache, and I agree we should figure out how to offer caching without having global side effects... but I'm still missing why exactly dask/dask-image#194 popped up because of it?

**Comment by GenevieveBuckley** [comment]
...nope! I don't remember any of the conversations @jni talks about. I'm just kicking this over here so it doesn't get lost completely.

**Comment by tlambert03** [comment]
ok that's fine.  I just wanted to clarify (for my own understanding) whether the cache was actually _causing_ that problem, or just came up as a topic while investigating that problem (and possibly made investigating it harder), but didn't actually contribute to the problem.  @jni, do you know?

**Comment by GenevieveBuckley** [comment]
> ...or just came up as a topic while investigating that problem (and possibly made investigating it harder), but didn't actually contribute to the problem

I'd lean towards this probably being a more accurate summary. Feel free to edit the text above if you like.

**Comment by jni** [comment]
@tlambert03 

Indeed, it just caused a red-herring, because calling `data.compute()` before adding the array to napari made the problem go away — but that's only because `dask.compute()` would cache the whole thing, so no more computation would need to happen. Discussion starts at this comment and the two that follow:

https://github.com/dask/dask-image/issues/194#issuecomment-786419669

**Comment by tlambert03** [comment]
gotcha.  thanks!

**Comment by GenevieveBuckley** [comment]
Thanks for the fix, @jni
